CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## 1.5.0 / 2017-01-23
+
+* [CHANGE] Use lexicographic order to sort alerts by name
+* [FEATURE] Add Joyent Triton discovery
+* [FEATURE] Add scrape targets and alertmanager targets API
+* [FEATURE] Add various persistence related metrics
+* [FEATURE] Add various query engine related metrics
+* [FEATURE] Add ability to limit scrape samples, and related metrics
+* [FEATURE] Add labeldrop and labelkeep relabelling actions
+* [FEATURE] Display current working directory on status-page
+* [ENHANCEMENT] Allow checkpoints and maintenance to happen concurrently
```

Comment:
According to the result of the lengthy discussion recently, I'd say this is way to internal to mention it here. Users won't even recognize what "maintenance" is. I would just leave it out. It's kind of covered by the "various performance improvements".